data-es6.js

```diff
@@ -2914,6 +2914,30 @@ exports.tests = [
   },
 },
 {
+  name: 'prototype of bound functions',
+  category: 'functions',
+  significance: 'small',
+  link: 'https://people.mozilla.org/~jorendorff/es6-draft.html#sec-boundfunctioncreate',
+  exec: function () {/*
+    function getProtoBound(proto) {
+      var f = Object.setPrototypeOf(function() { }, proto)
+      var boundF = Function.prototype.bind.call(f, null)
+      return Object.getPrototypeOf(boundF)
+    }
+
+    return [ Function.prototype, function(){}, [], null ].every(function(proto) {
```

Comment:
>Unless the tested implementation introduces a strange bug in its modified version of .bind()

Well, stranger bugs have happened. Consider the number of "instances of iterables" tests I've added to iterator consumer features - if they implemented the spec correctly, retrieving the `[Symbol.iterator]` method from the prototype chain should be no different than retrieving it directly. But, you can never be too sure. (In fact, I'm starting to wonder if I should split them all up into "generator instances" and "plain-object iterables" - [look at this in FF38](http://i.imgur.com/yGkJYPu.png)).